enterprise_access/apps/subsidy_request/models.py

```diff
@@ -160,3 +166,58 @@ def approve(self, reviewer_lms_user_id):
 
     def deny(self, reviewer_lms_user_id, reason):
         raise NotImplementedError
+
+
+class SubsidyRequestCustomerConfiguration(TimeStampedModel):
+    """
+    Stores subsidy_requests configuration for a customers
+
+    .. no_pii: This model has no PII
+    """
+
+    enterprise_customer_uuid = models.UUIDField(
+        primary_key=True,
+    )
+
+    subsidy_requests_enabled = models.BooleanField(
+        default=False,
+        help_text=_(
+            "Whether or not subsidy requests are enabled for an enterprise."
+        )
+    )
+
+    subsidy_type = models.CharField(
+        max_length=32,
+        blank=True,
+        null=True,
+        choices=SubsidyTypeChoices.CHOICES,
+        help_text=("Which type of subsidy is used to grant access."),
+    )
+
+    pending_request_reminder_frequency = models.CharField(
+        max_length=32,
+        blank=False,
+        null=False,
```

Comment:
Can we add a default in here?  My vote would be for weekly?

Reply:
actually i wonder, give this feature isn't technically required in the MVP, should default be never? (until we're ready to release this feature)